seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-doris/src/main/java/org/apache/seatunnel/flink/doris/sink/DorisOutputFormat.java

```diff
@@ -207,7 +207,7 @@ public synchronized void flush() throws IOException {
                 result = OBJECT_MAPPER.writeValueAsString(batch);
             }
         } else {
-            result = String.join(this.lineDelimiter, batch.toArray(new CharSequence[batch.size()]));
+            result = String.join(this.lineDelimiter, batch.toArray(new CharSequence[0]));
```

Comment:
I don't think this change will get a better performance. 
1. If we use `new CharSequence[0]`, it will create a extra Object array with length `batch.size()` and copy the element from batch to new array. 
2. If we use `new CharSequence[batch.size()]`, it will just copy the element from batch to the given array.
```java
public <T> T[] toArray(T[] a) {
    if (a.length < size)
        // Make a new array of a's runtime type, but my contents:
        return (T[]) Arrays.copyOf(elementData, size, a.getClass());
    System.arraycopy(elementData, 0, a, 0, size);
    if (a.length > size)
        a[size] = null;
    return a;
}
```
I don't think this change is necessary unless you can provide a benchmark test result.

Reply:
I read the code, and you're right. Let's close the PR. @ruanwenjun

Reply:
@legendtkl Maybe we need to add a benchmark module, then we can easy to test this kind of case. https://github.com/apache/kafka/tree/trunk/jmh-benchmarks